lib/index.js

```diff
@@ -100,6 +100,33 @@ module.exports.createFromSession = function (session) {
   }
 }
 
+module.exports.koaMiddlewhere = function (tid, options) {
+  this.tid = tid;
+  this.options = options;
+
+  var cookieName = (this.options || {}).cookieName || "_ga";
+
+  return async function (ctx, next) {
+    ctx.request.visitor = module.exports.createFromSession(ctx.session);
+    if (ctx.request.visitor) {
+      return await next();
```

Comment:
No need to await promise before return when function is async.

```suggestion
      return next();
```